### ISSUE TITLE
Error when assembling chunks, status code 504”

### ISSUE THREAD

**Issue by EgonHeuson** [nextcloud/all-in-one#978]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
Stock installation. Nothing was changed from AIO install.

### Actual behavior  
I just installed the latest version of Nextcloud AIO (I already had a manually installed version running). When I try to upload large files, I get the error “Error when assembling chunks, status code 504”. I’ve read in several places that this is related to the Apache (and/or nginx) timeout option, but while I knew very well how to change this option in my manually installed configuration, I confess I have no idea how to do it with Nextcloud AIO (and docker more generally). I think I “just” need to change the timeout in the Apache config, but if someone can help me by explaining how to do it properly, especially because this multi-container system seems sensitive to external modification. In particular how to restart Apache after the modification, etc? So if anyone can help me, that’s cool.).

**Comment by szaimen** [comment]
Hi, thanks for your report! How large is the file and how long did the upload take?

**Comment by EgonHeuson** [comment]
One of 604MB and the other of 2.7GB. I tried to zip them to check if it could come from the file format.
P.S. Woaw, what a fast answer ☺️

**Comment by szaimen** [comment]
Yes, Feel free to send them over. Also please answer these questions:

How large is the file and how long did the upload take?
Also, are you running AIO behind a reverse proxy?

**Comment by EgonHeuson** [comment]
Here is a link to download one of them from my other nextcloud instance (manual install): https://nextcloud.heuson.fr/s/yMmyF9LprKF6d4G
I upload at 5MB/sec in average and it take 2-3 min for this file.
Yes I am behind a Nginx Proxy Manager hosted on a separate VM on my proxmox. I used the following config as found in the Nextcloud AIO documentation:

<img width="502" alt="2022-07-28_21-42-39" src="https://user-images.githubusercontent.com/32910291/181624105-ac165437-f584-4239-9df6-ce1ee11f963b.png">

**Comment by szaimen** [comment]
I cannot reproduce this even with the provided video on a native AIO instance (without reverse proxy).
cc @2020Sanoj can you reproduce this?

**Comment by szaimen** [comment]
@EgonHeuson I suppose you tried to upload the file using the web interface while being logged in?

**Comment by EgonHeuson** [comment]
You want me to take a video of it? I did'nt change anything in the settings except installing the audio player. I was transfering my data. You want me to add here my NC config file? 
Ah yes I forgot to answer. I uploaded everything from web interface logged in.

**Comment by EgonHeuson** [comment]
Of course I could directly transfer it to the data older manually and rescan if needed, but that wouldn't be very practical for my other users :-(

**Comment by szaimen** [comment]
> Ah yes I forgot to answer. I uploaded everything from web interface logged in.

yes, this is what I did as well and it worked for me.

**Comment by EgonHeuson** [comment]
Hmmm. What's strange is that I get the error but I do still have the file uploaded apparently after I reload the page. It seems that it's 1) a false error, and 2) that it takes more time for the file to appear on the screen. Any idea?

**Comment by EgonHeuson** [comment]
Also I activated Collabora, and more importantly, ClamAV. Do you think it is too much for my NUC?

**Comment by EgonHeuson** [comment]
It seems to me that when I manually installed my first Nextcloud instance, I also had the problem, and I just had to increase the limits in the Apache config files. But here with Docker and the AIO version I don't really know how to do it :-(

**Comment by szaimen** [comment]
which limits did you increase?

**Comment by szaimen** [comment]
I still cannot reproduce this even with clamav enabled, btw...

**Comment by EgonHeuson** [comment]
> which apache limits did you increase?

Last time? I need to check. But I think something like in this post: https://github.com/nextcloud/server/issues/17992#issuecomment-1101942091

**Comment by szaimen** [comment]
> Last time?

yes

**Comment by EgonHeuson** [comment]
I followed this post: https://github.com/nextcloud/server/issues/17992#issuecomment-1101942091

**Comment by szaimen** [comment]
I see. The only thing that may make a difference is this:
`'tempdirectory' => '/var/big_temp_file/'`
`upload_tmp_dir = /var/big_temp_file/`

However this does not explain why it works on my test instance. 

Moving to discussion for now where we can discuss this further.
